compiler/rustc_target/src/spec/targets/aarch64_apple_darwin.rs

```diff
@@ -1,11 +1,12 @@
 use crate::spec::base::apple::{macos_llvm_target, opts, Arch};
-use crate::spec::{FramePointer, SanitizerSet, Target, TargetOptions};
+use crate::spec::{FramePointer, SanitizerSet, StackProbeType, Target, TargetOptions};
 
 pub fn target() -> Target {
     let arch = Arch::Arm64;
     let mut base = opts("macos", arch);
     base.cpu = "apple-m1".into();
     base.max_atomic_width = Some(128);
+    base.stack_probes = StackProbeType::Inline;
```

Comment:
Nit: would it be better to move some of this into the "base" options? For example, all `aarch64-apple-*` targets set this option so it could be moved into `rustc_target::spec::base::apple::opts` when `arch == Arch::Arm64`.

Reply:
Yeah, that sounds nicer -- I'll update it.

Reply:
OK, I've done this for `apple::opts` targets. Sadly, other targets don't have an `enum Arch` like that (yet) for us to do the same elsewhere, but it would be nice.